src/test/kotlin/carracing/game/domain/data/RaceTest.kt

```diff
@@ -0,0 +1,77 @@
+@file:Suppress("ktlint:standard:no-wildcard-imports")
+
+package carracing.game.domain.data
+
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.*
+
+class RaceTest {
+    private val carsFactory = mock<CarsFactory>()
```

Comment:
I believe unit tests for domain models should be designed so they can be thoroughly tested without using mock objects. So, how can `Race` be tested without mocks? The hint lies in the constructor.